pkg/registry/registry.go

```diff
@@ -177,7 +177,7 @@ func registerDriverFile(path string) {
 			name = strings.TrimSuffix(strings.TrimPrefix(base, "lima-driver-"), ".exe")
 		}
 	} else {
-		if strings.HasPrefix(base, "lima-driver-") {
+		if strings.HasPrefix(base, "lima-driver-") && !strings.HasSuffix(base, ".exe") {
```

Comment:
May I suggest creating a separate PR for this fix?

Reply:
Can be also a separate commit in the same PR